docs/content/configuration/caching.md

```diff
@@ -84,7 +84,8 @@ Uses memcached as cache backend. This allows all nodes to share the same cache.
 |`druid.cache.maxObjectSize`|Maximum object size in bytes for a Memcached object.|52428800 (50 MB)|
 |`druid.cache.memcachedPrefix`|Key prefix for all keys in Memcached.|druid|
 |`druid.cache.numConnections`|Number of memcached connections to use.|1|
-
+|`druid.cache.protocol`|Memcached communication protocol. Can be binary or text.|binary|
+|`druid.cache.locator`|Memcached locator. Can be consistent or array_mod.|consistent|
```

Comment:
Valid values - consistent, array_mod and vbucket.

Reply:
https://github.com/druid-io/druid/blob/c0beb78ffdb5807ceea8a340838eab01d6a18d1d/server/src/main/java/io/druid/client/cache/MemcachedCustomConnectionFactoryBuilder.java#L62

Reply:
For native spymemcached `ConnectionFactoryBuilder` it's invalid too.

Reply:
cool, thanks.